Signal/src/ViewControllers/HomeView/ConversationSearchViewController.swift

```diff
@@ -157,8 +157,9 @@ class ConversationSearchViewController: UITableViewController {
 
             let thread = searchResult.thread
             SignalApp.shared().presentConversation(for: thread.threadRecord,
-                                                   action: .compose,
```

Comment:
Note: I changed this one - I think when you are searching for a particular message you are usually not intending to compose a response. WDYT?

(I did leave the compose action for the "conversation" and "contact" results)